src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        Boolean fullyGuess = result1 == bet1 && result2 == bet2;
```

Comment:
Do we need these variables here? see checklist https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html